{"errors":[{"id":null,"links":null,"status":"500","code":"500-Cse-Internal-Server-Exception","title":"CSE run failed","detail":"CSE run failed; nested exception is java.lang.RuntimeException: The task has been interrupted","source":null,"meta":null}]}